doc/source/index.rst

```diff
@@ -0,0 +1,17 @@
+gcovr
+=====
+
+.. include:: ../../README.rst
+   :start-after: .. begin abstract
+   :end-before: .. end abstract
+
+.. toctree::
+   :maxdepth: 2
+   :numbered:
+   :caption: Contents:
+
+   installation
+   quickstart
+   contributing
+   license
+   guide
```

Comment:
For the initial switch to Sphinx-based docs, I'd initially avoid separate chapters for installation & quickstart since there are similar sections in the guide. We can later refactor that.

Since the guide is the most important part, I'd put it first:

 - guide
 - contributing
 - (changelog?)
 - license

What do you think of this structure?